crates/primitives/starknet/src/transaction/types.rs

```diff
@@ -269,6 +279,8 @@ impl From<Transaction> for InvokeTransaction {
 )]
 #[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
 pub struct Transaction {
+    /// The type of the transaction.
+    pub _type: TxType,
```

Comment:
May `_type` be `tx_type`? Rust reserved `_` for unused variables.
It might be a bit redundant, I agree.